viking/src/tools/check.rs

```diff
@@ -488,15 +497,67 @@ fn parse_args() -> Result<Args, lexopt::Error> {
         version,
         always_diff,
         update_matching,
+        print_help,
         other_args,
     })
 }
 
+fn print_help() -> Result<()> {
+    println!(
+"Usage: check [function name] [--version VERSION] [--update-matching] [--always-diff] [asm-differ arguments]
+    
+Checks if the compiled bytecode of a function matches the assembly found within the game elf. If not, show the differences between them.
+If no function name is provided, all functions within the respository function list will be checked.
+
+optional arguments:
+
+ -h, --help             Show this help message and exit
+ --version VERSION      Check the function against version VERSION of the game elf
+ --update-matching      If a function does match, but is marked as mismatching, mark it as matching in the function list
```

Comment:
```suggestion
 --update-matching      If a function does match, but is not marked as matching, mark it as matching in the function list
```
(for instance, this option also updates functions that are marked as not decompiled)